lumicks/pylake/kymo.py

```diff
@@ -184,6 +185,23 @@ def line_timestamp_ranges(self, exclude=None, *, include_dead_time=None):
 
         return self._line_timestamp_ranges_factory(self, not include)
 
+    def _tiff_frames(self, iterator=False) -> npt.ArrayLike:
+        """Create frames of TIFFs used by `export_tiff().`"""
+        return np.expand_dims(self.get_image(), axis=0)
+
+    def _tiff_image_metadata(self) -> dict:
+        """Create metadata for the ImageDescription field of TIFFs used by `export_tiff()`."""
+        metadata = super()._tiff_image_metadata()
+        # Add `Kymo` specific metadata
+        metadata["Line time (s)"] = self.line_time_seconds
+        return metadata
+
+    def _tiff_timestamp_ranges(self) -> list:
+        """Create Timestamp ranges for the DateTime field of TIFFs used by `export_tiff`."""
+        # As `Kymo` has only one frame, return a list with one timestamp range
```

Comment:
Actually, including extra metadata is fairly easy. I included 'Line timestamp ranges' in the method `_tiff_image_metadata()` for the kymo.

Reply:
yeah the problem isn't including extra information, I think we should spend some time planning it out though to avoid having a million different versions when we forget something etc. maybe let's set up a time for the 3 of us to look over everything and discuss?

Reply:
Change of plans: Instead of all `line timestamp ranges` there are now only two entries `Start pixel timestamp (ns)` and `Stop pixel timestamp (ns)`. With these and the number of lines the `Line timestamp ranges` can be reconstructed.